deploy-service/common/src/main/java/com/pinterest/deployservice/db/DBAgentDAOImpl.java

```diff
@@ -110,10 +107,8 @@ public void update(String hostId, String envId, AgentBean agentBean) throws Exce
     @Override
     public void updateMultiple(Collection<String> hostIds, String envId, AgentBean agentBean) throws Exception {
         SetClause setClause = agentBean.genSetClause();
-        String hostStr = QueryUtils.genStringGroupClause(hostIds);
-        String clause = String.format(UPDATE_AGENTS_BY_HOSTIDS, setClause.getClause(), hostStr);
-        setClause.addValue(envId);
-        new QueryRunner(dataSource).update(clause, setClause.getValueArray());
+        String clause = String.format(UPDATE_AGENTS_BY_HOSTIDS, setClause.getClause());
+        new QueryRunner(dataSource).update(clause, setClause.getValueArray(), hostIds, envId);
```

Comment:
I don't think this will work. `hostIds` is `Collection<String>` and the expectation is a `String`.